packaging/functions.sh

```diff
@@ -153,7 +153,7 @@ install_windows_launcher()
 	rm -rf "${SRC_PATH}/OpenRA.WindowsLauncher/obj"
 	dotnet publish "${SRC_PATH}/OpenRA.WindowsLauncher/OpenRA.WindowsLauncher.csproj" -c Release -r "${TARGETPLATFORM}" -p:LauncherName="${LAUNCHER_NAME}" -p:TargetPlatform="${TARGETPLATFORM}" -p:ModID="${MOD_ID}" -p:DisplayName="${MOD_NAME}" -p:FaqUrl="${FAQ_URL}" -o "${DEST_PATH}" --self-contained true
 
-	# NET 5 is unable to customize the application host for windows when compiling from Linux,
+	# NET 6 is unable to customize the application host for windows when compiling from Linux,
```

Comment:
> /usr/share/dotnet/sdk/6.0.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(550,5): warning NETSDK1074: The application host executable will not be customized because adding resources requires that the build be performed on Windows (excluding Nano Server). [/home/paul2/src/OpenRA/OpenRA.WindowsLauncher/OpenRA.WindowsLauncher.csproj]

Still true.